src/LoRa.cpp

```diff
@@ -607,6 +607,32 @@ void LoRaClass::setOCP(uint8_t mA)
   writeRegister(REG_OCP, 0x20 | (0x1F & ocpTrim));
 }
 
+void LoRaClass::setGain(uint8_t gain)
+{
+  // check allowed range
+  if(gain > 6) {
+	  gain = 6;
```

Comment:
Nitpick: The whitespace seems a bit off here and below. Maybe you are using tabs instead of spaces?

Reply:
Ok, I will fix it today.
Yes, I use tabs.

Reply:
Done at https://github.com/sandeepmistry/arduino-LoRa/pull/408/commits/42565206b85e2e43ed61151c8f192cdc073fbc9b